_layouts/page.html

```diff
@@ -1,21 +1,13 @@
 {% include default.html %}
 
-
 <body>
-  {% include nav-bar.html %}
-  <br><br><br>
+  {% include nav-bar2.html %}
   <div class="container">
     <div class='row'>
-      <div class='col-sm-8'>
-        <br><br><br>
-            {{ content }}
-      </div>
-      <div class='col-sm-3 col-sm-offset-1'>
-        <h4>BLOG</h4>
-        {% include side-blog.html %}
+      <div class='col-sm-12'>
+        {{ content }}
       </div>
     </div>
   </div>
-
 </body>
-{% include footer.html %}
+{% include footer2.html %}
```

Comment:
seems we should just delete the original ``footer.html``?

Reply:
⬆️